### PR TITLE
bugfix/whitespace in root

### LINKED ISSUE #2: Exception on loading a UserParameterReader within a whitespace-containing Vishnu root directory
If you run Vishnu.exe in a directory, whose name contains whitespaces and you have a UserParameterReader configured,
for example:
d:\Daten\\**Vishnu Root**\\ReadyBin\Vishnu.bin\Vishnu.exe --UserParameterReaderPath
="%UserAssemblyDirectory%\DemoParameterProvider.dll|Para-001|Para-002|M:1",
Vishnu will die with
`System.ApplicationException: Die User-Assembly d:\Daten\Vishnu konnte nicht geladen werden.`

### PR DESCRIPTION
**Pull request by netetireallyhuman**
- **BugFix: fixes #2: A UserParameterReader from within a whitespace-containing root-directory throwed an exception.**
- **Vishnu Version to 8.2.0.4.**

### CODE REVIEW

**Review by NetEtiWorker — APPROVED** [review]
Now everything works perfectly.
In my opinion, you can merge.😁👍

**Review by NetEtiWorker — APPROVED** [review]
👍

File: Vishnu.Interchange/AppSettings.cs
Comment (NetEtiWorker) [inline Vishnu.Interchange/AppSettings.cs:1]: 👍

**Review by NetEtiWorker — APPROVED** [review]
👍

**Review by NetEtiWorker — APPROVED** [review]
go

**Comment by NetEtiWorker** [comment]
go!

**Review by NetEtiWorker — APPROVED** [review]

**Comment by netetireallyhuman** [comment]
I messed it up with two users😒
Hope next time works better.

**Pull request closed without merging**